lib/diagnostic.rb

```diff
@@ -74,7 +91,12 @@ def example_diff
   end
 
   describe 'PATCH update' do
-    # your test(s) here
+    before(:each) do
+      patch :update, id: example.id
+
+  it 'is succesful and returns JSON object'
+    expect(response.status).to eq(200)
+    expect(JSON.parse(response.body)).not_to be_nil
```

Comment:
Just like with POST, make sure you're sending data to update at whichever `example.id` you are updating when PATCHing.

```
  describe 'PATCH update' do
    before(:each) do
      patch :update, id: example.id, example: example_diff, format: :json
    end

    it 'is successful' do
      expect(response).to be_successful
    end

    it 'renders a JSON response' do
      example_response = JSON.parse(response.body)
      expect(example_response).not_to be_nil
    end
  end
```